assets/js/templates/sidebar-items.handlebars

```diff
@@ -1,50 +1,58 @@
 <ul id="full-list">
-{{#each nodes as |node nodeId|}}
+  {{#each nodes as |node nodeId|}}
     {{#newGroup ../this node.group}}
       <li class="group">{{node.group}}</li>
     {{/newGroup}}
-    <li class="{{#isLocal node.id}}clicked open{{/isLocal}}">
 
-    {{!-- When visiting a module page, the link to this module page
-    in the menu should not link to a new page, instead should link
-    to the top of the page itself. --}}
-    {{#isEmptyNode node}}
-      <a href="{{node.id}}.html" title="{{node.title}}">{{node.title}}</a>
-    {{else}}
-      <a href="{{node.id}}.html" title="{{node.title}}" class="expand">{{node.title}}</a>
+    {{#nestingContextChanged ../this node.nested_context}}
+      <li class="nesting-context" title="{{node.nested_context}}" aria-hidden="true">{{node.nested_context}}</li>
+    {{/nestingContextChanged}}
```

Comment:
Sorry for the super minor nitpick but can we please keep the `newGroup` and `nestingContextChange` function names in sync? We can either call both `groupChanged` and `nestingChanged` or `newGroup` and `newNesting`, I would just like to keep them in sync. :)

Reply:
👍  I'll update them to `...Changed`. I didn't name it `newNesting` because it's confusing: the nesting isn't necessarily "new" as there can be duplicates.